src/release/breaking-changes/wide-gamut-framework.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Migration guide for wide gamut Color
+description: >-
+  Changes to support wide gamut color and migration instructions.
+---
+
+## Summary
+
+The API for the [`Color`][] class in `dart:ui` is changing to support [wide
+gamut color spaces][].
+
+## Context
+
+The Flutter engine [already supports wide gamut color][] with [Impeller][], and the support is now being added [to the framework][].
+
+The iOS devices that Flutter supports render to a larger array of colors,
+specifically in the [DisplayP3][] color space. After this change, the Flutter
+framework can render all of those colors on iOS Impeller, and the `Color` class
+will be better prepared for future color spaces or changes to
+color component bit depth.
+
+## Description of change
+
+Changes to [`Color`][]:
+
+ 1. Adds an enum field that specifies its [`ColorSpace`][].
+ 1. Adds API to use normalized floating-point color components.
+ 1. Removes API that uses 8-bit unsigned integer color components that can lead
+    to data loss.
+
+Changes to [`ColorSpace`][]:
+
+ 1. Add a `displayP3` property.
+
+## Migration guide
+
+### 8-bit unsigned integer constructors
+
+Constructors like `Color.fromARGB` remain unchanged and have continued
+support. To take advantage of Display P3 colors, you must
+use the new `Color.from` constructor that takes normalized floating-point
+color components.
+
+```dart
+// Before
+final magenta = Color.fromARGB(0xff, 0xff, 0x0, 0xff);
+// After
+final magenta = Color.from(alpha: 1.0, red: 1.0, green: 0.0, blue: 1.0)
+```
+
+### Implementors of `Color`
+
+There are new methods being added to `Color` so any class that `implements`
+Color will break and have to implement the new methods like `Color.a`, and
+`Color.b`. Ultimately implementors should be migrated to take advantage of the
+new API. In the short-term, these methods can easily be implemented without
+changing the underlying structure of your class.
+
+For example:
+
+```dart
+class Foo implements Color {
+  int _red;
+
+  @override
+  double get r => _red * 255.0;
+}
+```
+
+{{site.alert.note}}
+  The hope is to eventually make Color `sealed`. Now might be a good opportunity
+  to stop implementing `Color`. Instead of inheritance, switch to composition.
+{{site.alert.end}}
+
+### Color space support
+
+Clients that use `Color` and perform any sort of calculation on the color
+components should now first check the color space component before performing calculations. 
+To help with that, you can use the new `Color.withValues` method
+to perform color space conversions.
+
+For example:
+
+```dart
+// Before
```

Comment:
I think this is clear if you read the preceding paragraph, but maybe consider making it overly obvious why this is bad.

Maybe:

```suggestion
// BAD - Wrong if `x` is SRGB and `y` is DisplayP3 
```

Reply:
Done, I spelled in out in prose below.